Source/cmGlobalFastbuildGenerator.h

```diff
@@ -55,6 +55,12 @@ class cmGlobalFastbuildGenerator
 	virtual void ComputeTargetObjectDirectory(cmGeneratorTarget*) const;
 	virtual const char* GetCMakeCFGIntDir() const;
 
+	virtual void GetTargetSets(TargetDependSet& projectTargets,
+							   TargetDependSet& originalTargets,
+							   cmLocalGenerator* root, GeneratorVector const&);
+
+	const std::vector<std::string> & GetConfigurations() const;
```

Comment:
Not technically, at least not in C++03.
From http://www.open-std.org/jtc1/sc22/wg21/docs/cwg_defects.html#45
>...
>and 11.7 [class.access.nest] paragraph 1:
>The members of a nested class have no special access to members of an enclosing class,
>nor to classes or functions that have granted friendship to an enclosing class; the usual
>access rules (clause 11 [class.access] ) shall be obeyed. [...]
>...
It seems this was changed in c++11, but clang warns when compiling with c++03 support show this up.